activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -140,7 +144,7 @@ def supports_json?
       end
 
       def supports_multi_insert?
```

Comment:
I think we can just delete this method and leave the inherited one. And as we're removing the caller, I think we should deprecate the method in the abstract adapter.